code/go/0chain.net/blobber/node.go

```diff
@@ -0,0 +1,37 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+	"os"
+
+	"github.com/0chain/blobber/code/go/0chain.net/core/encryption"
+	"github.com/0chain/blobber/code/go/0chain.net/core/logging"
+	"github.com/0chain/blobber/code/go/0chain.net/core/node"
+	"go.uber.org/zap"
+)
+
+func setupNode() error {
+	fmt.Print("[5/10] setup blobber")
```

Comment:
Can we move this steps line into main.go where all these functions are being called? It is very ambiguos to have steps in the function themselves

Reply:
It was on `main`. but it seems is not good. so I move fmt to these functions. if you thing it is better on main,I will get them back.